### PR TITLE
Issue #1410

### PR DESCRIPTION
**Pull request by j0rt3g4**
Solving Issue #1410 
@JohanFreelancer9 This one needs to be checked please, thank you in advance

### CODE REVIEW

**Comment by j0rt3g4** [comment]
Solving issue #1410

File: Exchange/ExchangeOnline/mail-flow-best-practices/use-directory-based-edge-blocking.md [hunk 1]
```diff
@@ -68,8 +68,8 @@ If a message is sent to a valid email address in Office 365, the message continu
 
 **Notes**:
 
-- Until all of your valid recipients have been added to Exchange Online and replicated through the system, you should leave the accepted domain configured as **Internal relay**. Once the domain type has been changed to **Authoritative**, DBEB is designed to allow any SMTP address that has been added to the service (except for mail-enabled public folders). There might be infrequent instances where recipient addresses that do not exist in your Office 365 organization are allowed to relay through the service.
-
+- Until all of your valid recipients have been added to Exchange Online and replicated through the system, you should leave the accepted domain configured as **Internal relay**, specialy if you have Mail-Enabled Public Folders in the environment.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/mail-flow-best-practices/use-directory-based-edge-blocking.md]:
```suggestion
- Until all of your valid recipients have been added to Exchange Online and replicated through the system, you should leave the accepted domain configured as **Internal relay**, especially if you have mail-enabled public folders in the environment.
```

File: Exchange/ExchangeOnline/mail-flow-best-practices/use-directory-based-edge-blocking.md [hunk 2]
```diff
@@ -68,8 +68,8 @@ If a message is sent to a valid email address in Office 365, the message continu
...
+- You only do the change to **Authoritative**, once all the Public Folders and users are migrated to Exchange Online. DBEB is designed to allow any SMTP address that has been added to the service (except for mail-enabled public folders). There might be infrequent instances where recipient addresses that do not exist in your Office 365 organization are allowed to relay through the service.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/mail-flow-best-practices/use-directory-based-edge-blocking.md]:
```suggestion
- Only change the domain to **Authoritative** once all the public folders and users are migrated to Exchange Online. DBEB is designed to allow any SMTP address that has been added to the service (except for mail-enabled public folders). There might be infrequent instances where recipient addresses that do not exist in your Office 365 organization are allowed to relay through the service.
```

**Comment by scanum** [comment]
@mattpennathe3rd PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request closed without merging**